weaviate/collections/classes/config_named_vectors.py

```diff
@@ -319,10 +322,56 @@ def img2vec_neural(
     def multi2vec_clip(
         name: str,
         *,
+        vector_index_config: Optional[_VectorIndexConfigCreate] = None,
+        vectorize_collection_name: bool = True,
         image_fields: Optional[Union[List[str], List[Multi2VecField]]] = None,
         text_fields: Optional[Union[List[str], List[Multi2VecField]]] = None,
+        interference_url: Optional[str] = None,
+    ) -> _NamedVectorConfigCreate:
+        """Create a named vector using the `multi2vec_clip` model.
+
+        See the [documentation](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules/text2vec-gpt4all)
+        for detailed usage.
+
+        Arguments:
+            `name`
+                The name of the named vector.
+            `source_properties`
+                Which properties should be included when vectorizing. By default all text properties are included.
```

Comment:
Not required for this model, right?

Reply:
yep, they were actually included in all multi2vec modules, fixed that :)